posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title: "MicroProfile OpenAPI 2.0 plus new features and loads of improvements for Jakarta EE 9 in Open Liberty 20.0.0.11-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MicroProfile OpenAPI 2.0 plus new features and loads of improvements for Jakarta EE 9 in Open Liberty 20.0.0.11-beta - OpenLiberty.io
+seo-description: Open Liberty 20.0.0.11-beta comes with MicroProfile OpenAPI 2.0 which allows users to generate documentation directly from API's, plus two new features for Jakarta EE 9.
```

Comment:
Should "API's" have an apostrophe? Should it be just "APIs"?

Reply:
You're right, it should be APIs